languages/react-native/example/ios/Podfile

```
@@ -0,0 +1,43 @@
+target 'ReactNativeExample' do
+  config = use_native_modules!
+
+  use_react_native!(
+    :path => config[:reactNativePath],
+    # An absolute path to your application root.
+    :app_path => "#{Pod::Config.instance.installation_root}/.."
+  )
+
+
+  # Run Codegen during development for the example app.
+  pre_install do |installer|
```

RuboCop (1.75.5)

[warning] Unused block argument - `installer`. You can omit the argument if you don't care about it.

(Lint/UnusedBlockArgument)